flink-formats/flink-parquet/src/main/java/org/apache/flink/formats/parquet/row/ParquetRowDataWriter.java

```diff
@@ -189,6 +243,11 @@ public void write(RowData row, int ordinal) {
         public void write(RowData row, int ordinal) {
             recordConsumer.addBinary(Binary.fromReusedByteArray(row.getString(ordinal).toBytes()));
```

Comment:
`write(row.getString(ordinal))`? Others same.

Reply:
We don't have a `write(Object value)` method now